tests/TestModel.php

```diff
@@ -14,5 +14,10 @@ class TestModel extends Model
     protected $guarded = [];
     public $timestamps = false;
 
-    public $translatable = ['name', 'other_field'];
+    public $translatable = ['name', 'other_field', 'mutable_field'];
+
+    public function setMutableFieldAttribute($value, $lang = null)
+    {
+        $this->attributes['mutable_field'] = $value;
```

Comment:
`mutable` means something else than a field with a mutator.

Could you rename this to `field_with_mutator` everywhere?

Reply:
Sure

Reply:
@freekmurze it's done